### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Layrovell**

### CODE REVIEW

**Review by dtsehelnyk — APPROVED** [review]
хорошая работа)

File: src/transformState.js [hunk 1]
```diff
@@ -61,8 +61,27 @@
...
+        break;
+
+      case 'removeProperties':
+        for (const i of key.properties) {
+          delete state[i];
+        }
+        break;
+
+      case 'clear':
+        for (const i in state) {
```
Comment (varseniuk) [inline src/transformState.js:79]:
```suggestion
        for (const property of key.properties) {
          delete state[property];
        }
        break;

      case 'clear':
        for (const value in state) {
```

try to use more meaningful names for variables

File: src/transformState.js [hunk 2]
```diff
@@ -61,8 +61,27 @@
...
+          delete state[i];
+        }
+        break;
+    }
```
Comment (varseniuk) [inline src/transformState.js:83]: Add 'default' case

**Review by varseniuk — CHANGES_REQUESTED** [review]
Always try to add the 'default' case for 'switch...case' statements

**Review by varseniuk — APPROVED** [review]
On 'default case' you can log smth to console (like 'Error! Unknown type of data!').
Or just 'break' and stop the function

**Pull request closed without merging**